### PR TITLE
Feature/align with sketch

### PR DESCRIPTION
**Pull request by AndreasArledal**
- Simplified bootstrap variables.
  - Instead of having separate copy of bootstrap variables and another place of overrides/special vars everything is consolidated to just one version
  - Split variables into multiple files by type (ie typo, colors etc)

- Aligned with sketch files
  - Typography uses same values as in sketch
  - Colors are same as in sketch
  - Restructuring of colors

### CODE REVIEW

File: doc_assets/styleguide.scss [hunk 1]
```diff
@@ -101,16 +93,36 @@ body{
...
 }
 
-.blue-light {
-  background: $blue-light;
+.ui-color-lightest {
+  background: $ui-color-lightest;
+}
+
+.white {
+  background: #fff;
```
Comment (mnd-hound) [inline doc_assets/styleguide.scss]: Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: src/mnd-bootstrap/_variables.scss [hunk 2]
```diff
@@ -1,912 +1,13 @@
...
-//##
-
-//** Horizontal line color.
-$hr-border:                   $gray-lighter;
-
-//** Horizontal offset for forms and lists.
-$component-offset-horizontal: 180px;
-
-@import "variables_custom";
+@import 'bourbon/app/assets/stylesheets/bourbon';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 3]
```diff
@@ -1,912 +1,13 @@
...
+@import 'bootstrap-sass-twbs/assets/stylesheets/bootstrap/variables';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 4]
```diff
@@ -1,912 +1,13 @@
...
+
+@import 'variables/measures';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 5]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/colors';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 6]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/typography';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 7]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/tables';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 8]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/buttons';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 9]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/forms';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 10]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/navigation';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 11]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/progress_bars';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 12]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/overlays';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/_variables.scss [hunk 13]
```diff
@@ -1,912 +1,13 @@
...
+@import 'variables/alerts';
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Prefer double-quoted strings

File: src/mnd-bootstrap/variables/_colors.scss [hunk 14]
```diff
@@ -0,0 +1,122 @@
+//== Colors
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_colors.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_colors.scss [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+  <div class="color-box brand-warning">
+    <span>$brand-warning</span>
+  </div>
+  <div class="color-box brand-danger">
+    <span>$brand-danger</span>
+  </div>
+</div>
+ */
+
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_colors.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_colors.scss [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+  <div class="color-box text-color-dark">
+    <span>$text-color-dark</span>
+  </div>
+  <div class="color-box text-color-light">
+    <span>$text-color-light</span>
+  </div>
+</div>
+ */
+
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_colors.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_typography.scss [hunk 17]
```diff
@@ -0,0 +1,77 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_typography.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_typography.scss [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+---
+title: Typography
+name: typo
+category: CSS
+---
+In general we should not use typography classes but instead use the correct semantic elements which in almost all
+cases should have the correct styling.
+*/
+
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_typography.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_typography.scss [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+parent: typo
+---
+```html_example
+<h1>H1 <small>$font-size-very-large</small></h1>
+<h2>H2 <small>$font-size-larger</small></h2>
+<h3>H3 <small>$font-size-base</small></h3>
+```
+*/
+
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_typography.scss]: Use `//` comments everywhere

File: src/mnd-bootstrap/variables/_colors.scss (hunk 16 above)
Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_colors.scss]: Use `//` comments everywhere

Comment (mnd-hound) [inline src/mnd-bootstrap/variables/_colors.scss]: Use `//` comments everywhere

**Comment by jorkas** [comment]
KBK :shipit:

**Pull request merged**